test/unit/data_hygiene/policy_tagger_test.rb

```diff
@@ -0,0 +1,81 @@
+require "test_helper"
+require "data_hygiene/policy_tagger"
+
+class PolicyTaggerTest < ActiveSupport::TestCase
+  setup do
+    @content_id_1 = SecureRandom.uuid
+    @content_id_2 = SecureRandom.uuid
+    @published_edition = create(:published_publication)
+    @document = @published_edition.document
+    @draft_edition = create(:draft_publication, document: @document)
+    stub_registration
+  end
+
+  test "updates policies for all editions of the document" do
+    # adding
```

Comment:
These comments feel a little superfluous to me, but that's personal preference.

FWIW - I think the code is perfectly descriptive without them.